config/index.js

```diff
@@ -1,8 +1,8 @@
 const path = require('path');
```

Comment:
why is this file indexed?
should not be tracked

please update .gitignore if it is not listed there

Reply:
also it needs to contain production config for users who wants just launch cloned project

Reply:
Done